### ISSUE TITLE
document how federation would work

### ISSUE THREAD

**Issue by jedahan** [digicoop/kaiwa#19]
How could myfriend@theirdomain.com join/send a message to the kaiwa running on mydomain.com? Is that even possible?

**Comment by warcode** [comment]
The XMPP server, if set up properly, will automatically deal with any direct communication between you and the added contact if on a different server.

Uncertain if the MUCs can be accessed.

**Comment by jedahan** [comment]
Yeah it seems like direct communication is fine, but no way to MUC ....

**Comment by mwild1** [comment]
What problems are you having with MUC?

Do make sure you have a DNS entry for the MUC domain (which is required for
joining from remote servers).

**Comment by jedahan** [comment]
OK so I made two digitalocean instances. a.mydomain.com, and b.mydomain.com. user1@a.mydomain.com, user2@b.mydomain.com .

I can add users to each others user lists and chat with each other, but if I am user1@a.mydomain.com, and I try to add:

channelB@chat.b.mydomain.com
channelB@b.mydomain.com
channelB@chat.mydomain.com

I cannot send any messages across from one to the other

**Comment by sebu77** [comment]
The Prosody documentation may possibly help:
http://prosody.im/doc/chatrooms
http://prosody.im/doc/dns

**Comment by jedahan** [comment]
So I did docker -it sebu77/kaiwa-server and find /etc/prosody/ but nothing in that directory nor /app/ show the names of the rooms that I have as mucs, even though they show up in the web interface
